uiowa/uiowa
# Handle individual multisite exceptions thrown during the update process.

Resolves #1575 

With this change, it will no longer be necessary to re-deploy the same tag to production to "finish" deploying updates. All multisites will be processed and all errors logged in one job. 

Note that this does not change the behavior of `blt dsa`. That is a separate command that only runs locally.

### Testing
- Configure local multisites such that one fails before one that succeeds.
  - icsa.uiowa.edu
  - uiowa.edu
- Run `blt auda` which is what happens during a deployment.
- See that the first site fails but the process continues to second.
- See that the entire process is marked as a failure.
